packages/test/test-version-utils/src/test/minCompatVersion.spec.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { execSync } from "child_process";
+import { CompatKind } from "../../compatOptions.cjs";
+import { isCompatVersionBelowMinVersion } from "../compatConfig.js";
+
+describe("Minimum Compat Version", () => {
+	const allVersionsFromNpm = execSync(`npm show fluid-framework versions --json`, {
+		encoding: "utf-8",
+	});
+	const allVersions: string[] = JSON.parse(allVersionsFromNpm);
+	const latestVersion = allVersions[allVersions.length - 1];
+
+	it("bad min compat string", () => {
+		const invalidString = "invalid string";
+		try {
```

Comment:
You can use `assert.throws` instead of this try-catch block which is a lot cleaner.